### PR TITLE
Hotfix/single ui application

### PR DESCRIPTION
**Pull request by KostetskyiYaroslav**
-added `.vscode` foldre into `.gitignore`
-marged 3 other UI applications into signle
-changed reference from `/app/core/index.html` to `/app` into `index.html`

Following components doesn't work because of a previous routing.

### CODE REVIEW

**Comment by KostetskyiYaroslav** [comment]
Commit's names have went wrong because I use GitBash for work with Git and symbols like `.`  and ` are systematic.

File: MedicalDocManagment.WebUI/systemjs.config.js [hunk 1]
```diff
@@ -8,9 +8,9 @@
         // указываем загрузчику System, где искать модули
         map: {
             // locations of Angular applications
-            admin_app: 'app/admin',
-            login_app: 'app/core',
-            main_app: 'app/main',
+            //admin_app: 'app/admin',
+            //login_app: 'app/core',
+            //main_app: 'app/main',
```
Comment (AndriiKatsubo) [inline MedicalDocManagment.WebUI/systemjs.config.js:13]: Consider removing commented.

**Comment by AndriiKatsubo** [comment]
I had run the project and default web-page looked as following:
default
I think that you should fix this.

File: MedicalDocManagment.WebUI/app/index.html [hunk 2]
```diff
@@ -14,33 +14,36 @@
...
     <script src="node_modules/reflect-metadata/Reflect.js"></script>
     <script src="node_modules/systemjs/dist/system.src.js"></script>
     <script src="systemjs.config.js"></script>-->
-
     <!-- Angular2 Code -->
     <script>
-        System.import('main_app').catch(function (err) { console.error(err); });
+        //System.import('login_app').catch(function (err) { console.error(err); });
+        //System.import('admin_app').catch(function (err) { console.error(err); });
+        //System.import('main_app').catch(function (err) { console.error(err); });
```
Comment (AndriiKatsubo) [inline MedicalDocManagment.WebUI/app/index.html:38]: Consider removing commented System.import's.

**Review by AndriiKatsubo — CHANGES_REQUESTED** [review]
Changes were requested according to the result of running the project in it's current state.

**Comment by KostetskyiYaroslav** [comment]
I wish to do it but without routing it's impossible.
Alex, said that he is going to fix routing. After that I be able to fix this layout.

**Comment by AndriiKatsubo** [comment]
> I wish to do it but without routing it's impossible.
Alex, said that he is going to fix routing. After that I be able to fix this layout.

Ok then.

**Review by AndriiKatsubo — APPROVED** [review]

**Review by Ravennod — APPROVED** [review]

**Pull request merged**